lib/views/setting/setting_view.dart

```diff
@@ -36,127 +40,15 @@ class _SettingViewState extends State<SettingView> {
     return true;
   }
 
-  Widget _buildGrpcSettingView(
-      {required BuildContext context, required Setting setting}) {
-    return Wrap(
-        runSpacing: MediaQuery.of(context).size.height * 0.05,
-        children: <Widget>[
-          Row(
-            children: [
-              const Text("TLS certificate directory path"),
-              SizedBox(
-                width: MediaQuery.of(context).size.width * 0.1,
-              ),
-              ElevatedButton(
-                  onPressed: () async {
-                    await pickDir();
-                  },
-                  child: const Text('Browser')),
-            ],
-          ),
-          InputDecorator(
-            decoration: const InputDecoration(border: OutlineInputBorder()),
-            child: Text(setting.path ?? "not found"),
-          ),
-          const Text("Host"),
-          TextFormField(
-            controller: TextEditingController(text: setting.host ?? ''),
-            onChanged: (text) {
-              setting.host = text;
-            },
-            decoration: const InputDecoration(
-              border: OutlineInputBorder(),
-            ),
-          ),
-        ]);
-  }
-
-  Widget _buildUnixSettingView(
-      {required BuildContext context, required Setting setting}) {
-    return Wrap(
-        runSpacing: MediaQuery.of(context).size.height * 0.05,
-        children: <Widget>[
-          Row(
-            children: [
-              const Text("lightning-rpc file path"),
-              SizedBox(
```

Comment:
same

Reply:
I think it was because my main branch was not synced with the upstream